openage/convert/value_object/read/media/smx.pyx

```diff
@@ -49,8 +47,8 @@ class SMXLayerType(Enum):
     """
     SMX layer types.
     """
-    MAIN    = "main"
-    SHADOW  = "shadow"
+    MAIN = "main"
+    SHADOW = "shadow"
     OUTLINE = "outline"
```

Comment:
These spaces should not be formatted away. If autopep8 did that, you should configure its settings to exclude these changes. There's instructions in the docs on how to configure autopep8 in VSCode for our code style: https://github.com/SFTtech/openage/blob/master/doc/ide/vscode.md#opening-and-configuring-the-openage-workspace

Reply:
Also, you should probably not format Cython code with autopep8 as it doesn't understand a lot of Cython syntax.